src/sentry/web/forms/accounts.py

```diff
@@ -558,7 +560,8 @@ def __init__(self, project, user, *args, **kwargs):
         specified_email = UserOption.objects.get_value(user, project, 'mail:email', None)
         emails.extend([user.email, alert_email, specified_email])
 
-        choices = [(email, email) for email in set(emails) if email is not None]
+        choices = [(email, email) for email in set(emails) if email]
```

Comment:
nitpick: but we should probably reliably sort these. A set is completely unsorted, so this means the choices could bounce around between reloads. Could easily sort them. Just `sorted(set(emails)))`